metagpt/ext/werewolf/roles/base_player.py

```diff
@@ -86,6 +86,7 @@ async def _think(self):
             # FIXME: hard code to split, restricted为"Moderator"或"Moderator, 角色profile"
             # Moderator加密发给自己的，意味着要执行角色的特殊动作
             self.rc.todo = self.special_actions[0]()
+        return self.rc.todo is not None
```

Comment:
I recommend directly returning True for easier understanding. In the werewolf case, we always want to trigger the following _act once we enter _think

Reply:
> I recommend directly returning True for easier understanding. In the werewolf case, we always want to trigger the following _act once we enter _think

Ok, Changing this for all files.